App/Containers/Wallet.js

```diff
@@ -29,21 +26,18 @@ class Wallet extends React.PureComponent {
   static navigationOptions = ({ navigation }) => {
     const params = navigation.state.params || {}
 
-    const accountBackupEnabled = params.storage && params.storage.enableWalletBackup.status
-    const pinEnabled = params.storage && params.storage.autoPinPhotos.status
     const photoBackupEnabled = params.storage && params.storage.enablePhotoBackup.status
 
     const headerLeft = (
       <TextileHeaderButtons>
         <Item
           title='backup'
-          iconName='cloud'
+          iconName={photoBackupEnabled ? 'cloud-checked' : 'cloud'}
```

Comment:
Nice. You can also set the color, just pass a `color` prop set to brandBlue or whatever. If you want.

Reply:
actually didn't work... didn't dig deep into it but yeah, color param was there before and not working